### DISCUSSION TITLE
Automated Backups to Mega.nz and Nextcloud

### DISCUSSION THREAD

**Discussion by mugdhosapiens** [standardnotes/forum#2593] [Suggestions] [answered]
I know extended accounts have automated backups support of data to Email, Dropbox, OneDrive, and Google Drive. Except for the Email, others are privacy nightmare. I would love to get my backups in Nextcloud or Mega.

**Comment by atmoio** [comment, marked as answer]
Hopefully WebDAV support at some point will solve this problem.

**Comment by frego** [comment]
+1 for WebDAV support. It would be a proper way to handle pictures as well.

**Comment by CornMaster** [comment]
Just a +1 for syncing notes via webdav.  I'd love to have a backup to my Nextcloud server, just in case.

**Comment by SiGamma** [comment]
Just wondering if WebDAV support for CloudLink is still on the roadmap? I'm in the process of getting a NAS primarily for other reasons, but I was excited about also having automated SN backups saved directly to it. It turns out I misremembered and only FileSafe supports WebDAV.

**Comment by drohm** [comment]
Would love to see WebDAV support, hopefully its still on the table.

**Comment by MorgothSauron** [comment]
I'm considering switching from dropbox to nextcloud. It would be great to support other cloud storage solution that is not Dropbox, OneDrive, or Google Drive.

**Comment by myreli** [comment]
Relates to https://github.com/standardnotes/forum/discussions/2613
